module/Activity/src/Activity/Service/Activity.php

```diff
@@ -132,7 +133,7 @@ public function createUpdateProposal(ActivityModel $oldActivity, array $params,
             $em->remove($oldUpdate);
             $em->flush();
 
-            return $oldProposal;
+            return false;
```

Comment:
If there exists a previous proposal, this early return means that the current proposal cannot be applied immediately. However, an admin should be able to do this, as a result of this PR.